lms/urls.py

```diff
@@ -958,9 +934,6 @@
     urlpatterns += [
         url(r'', include('third_party_auth.urls')),
         url(r'api/third_party_auth/', include('third_party_auth.api.urls')),
-        # NOTE: The following login_oauth_token endpoint is DEPRECATED.
-        # Please use the exchange_access_token endpoint instead.
-        url(r'^login_oauth_token/(?P<backend>[^/]+)/$', student_views.login_oauth_token),
```

Comment:
I'm not seeing where this went.  Did you intentionally drop it?

Reply:
Yup. See above.